### PR TITLE
Add variable focus ring

### PR DESCRIPTION
**Pull request by dan-gamble**

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js [hunk 1]
```diff
@@ -10,6 +11,10 @@ import { externalLinks, iframeFix } from './utils'
 new Vue(App).$mount('#app')
 
 document.addEventListener('DOMContentLoaded', () => {
+  const body = document.body || document.documentElement
+  body.classList.add('util-JSEnabled')
+  body.classList.remove('usr-IsTabbing')
```
Comment (danielsamuels) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js]: You've used `util-IsTabbing` everywhere else, but `usr-` here.

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css [hunk 2]
```diff
@@ -7,6 +7,26 @@
...
+[role='button'],
+input:not([type='range']),
+label,
+select,
+summary,
+textarea {
+  /* stylelint-disable property-no-vendor-prefix */
+  -ms-touch-action: manipulation;
+  touch-action: manipulation;
+}
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css:24]: Is this related to the tabbing fix? I am not sure how.

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css:24]: (It's fine if it's not - I'm just curious if it is)

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css:24]: It's not but i added it as it was semi along the same lines. It's for focus/active on iOS

Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css:24]: Seems related enough to me :)

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js [hunk 3]
```diff
@@ -10,6 +11,10 @@ import { externalLinks, iframeFix } from './utils'
 new Vue(App).$mount('#app')
 
 document.addEventListener('DOMContentLoaded', () => {
+  const body = document.body || document.documentElement
+  body.classList.add('util-JSEnabled')
+  body.classList.remove('util-IsTabbing')
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js]: Perhaps move this line into `utils/focus-ring.js` as it seems that file is quite self-contained?

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/main.js]: For some reason i thought it would need to be in `DOMContentLoaded` but i think it could be moved there.

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css [hunk 4]
```diff
@@ -7,6 +7,26 @@
 | Global resets to make life easier
 |
 */
+a,
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css]: I think explaining this part of the file in a comment would be good - the other resets in the file have one.

Comment (dan-gamble) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css]: Sure thing 👍

**Review by lewiscollard — APPROVED** [review]
This works really well on Bradfield. :)
